source/common/common/version_linkstamp.cc

```diff
@@ -2,5 +2,5 @@
 extern const char build_scm_revision[];
 extern const char build_scm_status[];
 
-const char build_scm_revision[] = BUILD_SCM_REVISION;
-const char build_scm_status[] = BUILD_SCM_STATUS;
+const char build_scm_revision[] = STABLE_BUILD_SCM_REVISION;
```

Comment:
Curious if you think we can add a test to avoid regressions such as https://github.com/envoyproxy/envoy/issues/2551?

Reply:
Not that I know of. Maybe we could hack something together in CI to add a fake empty commit, relink, and run `envoy --version`? Seems kinda fragile.

Reply:
FYI: I reverted changes to this file so we don't need to rename SCM status keys in everything that might build Envoy as a dependency.